app/controllers/reservable_tables_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ReservableTablesController < ApplicationController
+  def new
+    @reservable_table = ReservableTable.new
+  end
+
+  def create
+    @reservable_table = current_planner
+                        .reservable_tables
+                        .build(reservable_table_params)
```

Comment:
メソッドチェーンの時は2スペあると良さそうです！
```suggestion
                          .reservable_tables
                          .build(reservable_table_params)
```

Reply:
対応しました。

Reply:
おろ、対応されてなさそうです 👀

Reply:
あ、すみません。rubocopが修正していたみたいです。
対応しました。